docs/minishift-installation/master.adoc

```diff
@@ -8,6 +8,10 @@ include::topics/templates/document-attributes.adoc[]
 
 This guide walks you through the process of installing the {launcher} application to run on a local cloud as provisioned by a {OpenShiftLocal}. This includes link:https://www.openshift.org/minishift/[{Minishift}], an all-in-one VM that includes a community version of OpenShift Origin, or link:https://developers.redhat.com/products/cdk/overview/[{CDK}], a VM that includes OpenShift Container Platform.
 
+.The {launcher} Application
+The {launcher} application runs on OpenShift and provides a hassle-free way of creating functional example applications called missions. When using the {launcher} application, you choose the mission you want to generate, the specific runtime you want to use, and how you want to build and run it. The {launcher} application then generates your custom project, called a booster, and either launches it directly to the same OpenShift instance or provides a downloadable ZIP version of the project.
```

Comment:
> and how you want to build and run it

This implies that you build and run the runtime, is that really the case? If not, I would suggest using the actual object here instead of _it._

> to the same OpenShift instance or provides

The _either_ and _or_ clauses are customarily separated by a comma.